### ISSUE TITLE
ctrl-cmd-down causes 'no action' sound effect to play even though it fires correctly

### ISSUE THREAD

**Issue by mikayla-maki** [zed-industries/zed#5083]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

1. Open a file with multiple text lines
2. use ctrl-cmd-down to move a line down
3. 💥 hear incorrect sound effect

### Environment

N/A

**Comment by krstp** [comment]
So quick update to your latest post:
- `CTRL+CMD+DOWN` does not seem to be valid anymore
- from what I understand similar functionality is provided now by `ALT+DOWN`. Functionality has been verified on MacOS and Linux boxes.
I hope this helps. Thanks for all the hard work you guys do!

**Comment by JosephTLyons** [comment]
Yeah, the bug still exists, but we no longer use that default to move a line down, at least in the default VS Code keymap.
